### DISCUSSION TITLE
Large files

### DISCUSSION THREAD

**Discussion by Hendrik1987** [DHI/modelskill#23] [Ideas]
Model outputs for sure, but also measurements can easily exceed memory. How is this addressed? Can intermediate results easily be saved / read? Is aggregation in chunks possible? This need increases when handling output from multiple models and/or including multi-dimensional data like wave spectra.

**Comment by ecomodeller** [comment]
There are many options:
You can do lot's of pre-processing and supply the data as xarray dataarray or pandas dataframe, that gives you some freedom.
You can save a Comparer or ComparerCollection to disk.
